upup/pkg/fi/nodeup/template_functions.go

```diff
@@ -281,3 +283,12 @@ func (t *templateFunctions) KubeProxyConfig() *api.KubeProxyConfig {
 
 	return config
 }
+
+func (t *templateFunctions) BuildAPIServerAnnotations() map[string]string {
+	annotations := make(map[string]string)
+	annotations["dns.alpha.kubernetes.io/internal"] = t.cluster.Spec.MasterInternalName
```

Comment:
*should have* 

Although I ran into a weird testing something or other the other day.. it's just scary because stuff sneaks into the code base..

We really need getters/setters..

I digress, this should be fine - we exercise the code enough in testing to catch this